src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -158,11 +202,41 @@ export class SkyTabsetComponent
   }
 
   public ngOnDestroy() {
+    this.ngUnsubscribe.next();
+    this.ngUnsubscribe.complete();
     this.tabsetService.destroy();
   }
 
   private updateDisplayMode(currentOverflow: boolean) {
     this.tabDisplayMode = currentOverflow ? 'dropdown' : 'tabs';
     this.changeRef.markForCheck();
   }
+
+  private watchQueryParamChanges(): void {
+    this.activatedRoute.queryParams
+      .distinctUntilChanged()
+      .takeUntil(this.ngUnsubscribe)
+      .subscribe((params) => {
+        const activeTabsetParam = params[this.queryParam];
+
+        if (!activeTabsetParam) {
+          this.tabsetService.activateTabIndex(this.active);
```

Comment:
I'm not quite following... maybe I'm missing something simple, but why do we need to activate a tab if the query param is NOT defined?

Reply:
Good eye; removed.